config/oba.yml

```diff
@@ -4,23 +4,42 @@ idspace: OBA
 base_url: /obo/oba
 
 products:
-- oba.owl: https://raw.githubusercontent.com/obophenotype/bio-attribute-ontology/master/oba.owl
-- oba.obo: https://raw.githubusercontent.com/obophenotype/bio-attribute-ontology/master/oba.obo
-- oba.json: https://raw.githubusercontent.com/obophenotype/bio-attribute-ontology/master/oba.json
+- oba.owl: https://github.com/obophenotype/bio-attribute-ontology/releases/latest/download/oba.owl
+- oba.obo: https://github.com/obophenotype/bio-attribute-ontology/releases/latest/download/oba.obo
+- oba.json: https://github.com/obophenotype/bio-attribute-ontology/releases/latest/download/oba.json
 
 entries:
+- exact: /subsets/oba-basic.owl
+  replacement: https://github.com/obophenotype/bio-attribute-ontology/releases/latest/download/oba-basic.owl
+- exact: /subsets/oba-basic.obo
+  replacement: https://github.com/obophenotype/bio-attribute-ontology/releases/latest/download/oba-basic.obo
 - prefix: /tracker/
   replacement: https://github.com/obophenotype/bio-attribute-ontology/issues
 - prefix: /subsets/
   replacement: https://raw.githubusercontent.com/obophenotype/bio-attribute-ontology/master/subsets/
+- prefix: /releases/2022-01-19/
+  replacement: https://raw.githubusercontent.com/obophenotype/bio-attribute-ontology/v2022-01-19/
+  tests:
+    - from: /releases/2022-01-19/oba.owl
+      to: https://raw.githubusercontent.com/obophenotype/bio-attribute-ontology/v2022-01-19/oba.owl
+- prefix: /releases/2021-12-03/
+  replacement: https://raw.githubusercontent.com/obophenotype/bio-attribute-ontology/v2021-12-03/
+  tests:
+    - from: /releases/2021-12-03/oba.owl
+      to: https://raw.githubusercontent.com/obophenotype/bio-attribute-ontology/v2021-12-03/oba.owl
+- prefix: /releases/201
+  replacement: https://raw.githubusercontent.com/obophenotype/bio-attribute-ontology/v201
+  tests:
+    - from: /releases/2017-01-16/oba.owl
+      to: https://raw.githubusercontent.com/obophenotype/bio-attribute-ontology/v2017-01-16/oba.owl
 - prefix: /releases/
   replacement: https://raw.githubusercontent.com/obophenotype/bio-attribute-ontology/v
```

Comment:
`/releases/` still points to a branch, not a GitHub release. Is that right?

Reply:
Very nice catch, thank you!

Reply:
(Updated it now)